litestar/handlers/asgi_handlers.py

```diff
@@ -101,4 +104,28 @@ async def handle(self, connection: ASGIConnection[ASGIRouteHandler, Any, Any, An
         await self.fn(scope=connection.scope, receive=connection.receive, send=connection.send)
 
 
-asgi = ASGIRouteHandler
+def asgi(
```

Comment:
is this public/documented?

Reply:
and a `.. versionadded::` directive :D

Reply:
Actually, I'm not so sure about the `versionadded` with these. The names aren't newly introduced, and their usage remains largely the same. The fact that they're now implemented as decorator functions instead of callable class instance is an implementation detail. I think it's fair to say this is merely a change and not addition of a new feature.

wdyt @JacobCoffee @peterschutt?